src/transformers/models/deformable_detr/configuration_deformable_detr.py

```diff
@@ -116,6 +116,8 @@ class DeformableDetrConfig(PretrainedConfig):
             based on the predictions from the previous layer.
         focal_alpha (`float`, *optional*, defaults to 0.25):
             Alpha parameter in the focal loss.
+        use_custom_kernel (`bool`, *optional*, defaults to `False`):
+            Whether to use custom CUDA kernel to speed up inference and training on GPU.
```

Comment:
As said in the thread, let's not use a config parameter for this but just try to use the fast version and default to the slow one if the fast one is failing. If a user sets it to True and then pushes their model, anyone not having a GPU or a PyTorch installed without CUDA will get a failure when using this model.

Config parameters are usually bad for flags that depend on hardware/setups.

Reply:
Ok, I don't think there needs to be any changes to the main branch then. Should I go ahead and close the PR?

Reply:
If it's all good on main, then yes :-)